src/app/settings/organization-plans.component.html

```diff
@@ -206,16 +206,22 @@ <h2 class="spaced-header">{{'summary' | i18n}}</h2>
             </label>
         </div>
         <hr class="my-3">
-        <div class="text-lg">
-            <strong>{{'total' | i18n}}:</strong> {{subtotal | currency:'USD $'}} /{{selectedPlanInterval | i18n}}
+        <h2 class="spaced-header mb-4">{{ (createOrganization ? 'paymentInformation' : 'billingInformation') | i18n}}</h2>
+        <app-payment *ngIf="createOrganization" [hideCredit]="true"></app-payment>
+        <app-tax-info (onCountryChanged)="changedCountry()"></app-tax-info>
+        <div id="price" class="my-4">
+            <div class="text-muted text-sm">
+                Plan price: {{ subtotal | currency: 'USD $' }}
+                <br />
+                <ng-container>
+                    Estimated tax: {{ taxCharges | currency: 'USD $' }}
```

Comment:
Is there a reason we've chosen "Estimated tax" vs. "Tax"?

Reply:
I modified Amazon's wording of "Estimated tax to be collected". "Estimated" seemed appropriate to me since there's no real way to guarantee our math and Stripe's are always 1:1 matches, even though they should be and always are in testing.